analyzer/src/main/kotlin/PackageManager.kt

```diff
@@ -300,7 +300,7 @@ abstract class PackageManager(
         definitionFiles.forEach { definitionFile ->
             val relativePath = definitionFile.relativeTo(analysisRoot).invariantSeparatorsPath.ifEmpty { "." }
 
-            logger.info { "Resolving $managerName dependencies for path '$relativePath'..." }
+            logger.info { "Resolving $managerName dependencies for definition file '$relativePath'..." }
```

Comment:
This change revealed to me that the `.ifEmpty { "." }` two lines doesn't really make sense. First of all, it should not happen, and secondly even if it was triggered "." would be no "definition file". Mind adding a commit that simply removes `.ifEmpty { "." }`?

Reply:
Ah, exactly... that's also the reason why this was more generally called "path" instead of "definition file". So should we keep it at "path"?

Reply:
Yes, I have dropped the commit.